### DISCUSSION TITLE
Email doesn't send

### DISCUSSION THREAD

**Discussion by tsirigwtis** [gophish/gophish#3128] [General]
Hi, there, i have setup a Linux vps with the Gophish, and although works when i try to send the email the button is spinning and the email doesn't send. Does anyone know what may have happened? 

![Screenshot 2024-04-04 at 00 14 53](https://github.com/gophish/gophish/assets/69371696/38a6312f-a0f4-4623-afc4-195cba569fcb)

Thanks in advance!

**Comment by afifosh** [comment]
check the console error in the cmd seems you are using gmail maybe u need to check send less secure try to search on this or i woule recommend to open a mailtrap account and try to send throw a mailtrap smtp account for testing purpsoes just to make sure that there is no issue with the gophish

**Reply by BigBoss465** [reply]
![Captura de tela 2024-06-24 204031](https://github.com/gophish/gophish/assets/173743213/619713fe-0a7f-45b6-a6b8-5eb4664cbd40)
